src/event.ts

```diff
@@ -168,6 +169,76 @@ export default class Event extends AinftBase {
       });
   }
 
+  getActivity({
+    appId,
+    userId,
+    eventId,
+    activityId,
+    createdAt,
+    options,
+  }: GetEventActivityParams) {
+    const timestamp = Date.now();
+    const query = {
+      appId,
+      userId,
+      activityId,
+      createdAt,
+      ...options,
+    };
+    const data = buildData(
+      HttpMethod.GET,
+      `/event/${eventId}/activity`,
+      timestamp,
+      query
+    );
+    const signature = this.signData(data);
+    return axios
+      .get(`${this.baseUrl}/${eventId}/activity`, {
+        params: query,
+        headers: {
+          'X-AINFT-Date': timestamp,
+          Authorization: `AINFT ${signature}`,
+        },
+      })
+      .then((res) => res.data)
+      .catch((error) => {
+        throw error?.response.data;
+      });
+  }
+
+  updateActivityStatus({
+    eventId,
+    activityId,
+    appId,
+    createdAt,
+    status,
+  }: UpdateEventActivityStatusParams) {
+    const timestamp = Date.now();
+    const body = {
+      appId,
+      status,
+      createdAt,
+    };
+    const data = buildData(
+      HttpMethod.PUT,
+      `/event/${eventId}/activity/${activityId}`,
+      timestamp,
+      body
+    );
+    const signature = this.signData(data);
+    return axios
+      .put(`${this.baseUrl}/${eventId}/activity/${activityId}`, body, {
+        headers: {
+          'X-AINFT-Date': timestamp,
+          Authorization: `AINFT ${signature}`,
+        },
+      })
+      .then((res) => res.data)
+      .catch((e) => {
+        throw e.response?.data;
```

Comment:
do we need the `?` for other errors as well?

Reply:
Maybe Yes...  
I forgot to start the server in local and experienced an error in that part while testing.
Even if the server dies, it seems necessary for the SDK to handle errors well.
Can I fix it in this PR? Or the next PR?

Reply:
I see! Let's fix it in the next PR (and possibly include it in the next next release, depending on the ETA of @ehgmsdk20 ?)

Reply:
Thanks for review~!